src/resources/locales/en.json

```diff
@@ -854,9 +854,229 @@
             "disclaimer": "Shortcuts were defined from QWERTY keyboard layout.",
             "editUserJson": "Edit user file...",
             "keyboardShortcuts": "Keyboard shortcuts",
+            "list": {
+                "AddBookmarkWithLabel": {
+                    "description": "",
+                    "name": "Add Bookmark with Label"
+                },
+                "AnnotationsCreate": {
+                    "description": "When a text is selected, it will be marked as annotation and the annotation dialog box will open to allow additional text and select of color and stroke.",
+                    "name": "Create Annotation"
+                },
+                "AnnotationsCreateQuick": {
+                    "description": "When a text is selected, it will be marked as annotation without opening the annotation dialog box.",
+                    "name": "Create Quick Annotation"
+                },
+                "AnnotationsToggleMargin": {
+                    "description": "Changes the Annotations marks from the text to the margin or reverse.",
+                    "name": "Toggle Annotation Margin"
+                },
+                "AudioNext": {
+                    "description": "in reader window, calls the \"next\" command for audio (TTS or EPUB3 Media Overlays readaloud, or \"forward\" for audiobooks).",
+                    "name": "Next Audio"
+                },
+                "AudioNextAlt": {
+                    "description": "in reader window, calls the \"next\" command for audio (TTS or EPUB3 Media Overlays readaloud, or \"forward\" for audiobooks).",
+                    "name": "Next Audio (Alternative)"
+                },
+                "AudioPlayPause": {
+                    "description": "in reader window, calls the \"play\" or \"pause\" toggle commands for audio (TTS or EPUB3 Media Overlays readaloud).",
+                    "name": "Play/Pause Audio"
+                },
+                "AudioPrevious": {
+                    "description": "in reader window, calls the \"previous\" command for audio (TTS or EPUB3 Media Overlays readaloud, or \"rewind\" for audiobooks).",
+                    "name": "Previous Audio"
+                },
+                "AudioPreviousAlt": {
+                    "description": "in reader window, calls the \"previous\" command for audio (TTS or EPUB3 Media Overlays readaloud, or \"rewind\" for audiobooks). Alternative shortcut key.",
```

Comment:
This is not really Alternative way, I think. `Alt` shortcut for audio previous/next enable escape boolean in `ttsNext` / `ttsPrevious` r2-nav function. see : https://github.com/readium/r2-navigator-js/blob/9724ef3d4523e956e5e616d91f9f10872b4c02a7/src/electron/renderer/webview/readaloud.ts#L434

@danielweck Maybe can you improve this shortcut description ?